### ISSUE TITLE
pView cannot bring bView to front after the computer wakes up

### ISSUE THREAD

**Issue by damithc** [HubTurbo/HubTurbo#850]
I noticed this three times, each time after the comp wakes up after a long sleep. Seems to be fine after I restart HT.

**Comment by damithc** [comment]
Happened once even without the computer going to sleep.

**Comment by damithc** [comment]
Happening quite frequently now and not always related to a sleep/wakeup. Possibly caused by the change @acruis did recently, to fix #848 ? I did not have this problem before.

**Comment by acruis** [comment]
Yep, it's most likely because of that change.

Turns out it's pretty easy to replicate this, too. Focus on a different window than bView/pView, then focus back to and minimize bView. After doing this, focusing on pView would not bring bView to front (because it's minimized).

In this context, the `user32.ShowWindow` call *does* do something. It restores the bView window from its minimized state, which in turn prepares bView to be brought to front by the `SetWindowPos` call following it.

I deleted the `user32.ShowWindow` call in #848 because it couldn't bring bView to front, but didn't consider that `user32.ShowWindow` was not meant to bring bView to front. The fix for this issue is to put back the call.
